AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -1,6 +1,6 @@
-# `Intel® TensorFlow* Model Zoo Inference With FP32 Int8` Sample
+# `Intel® Reference models TensorFlow* Inference With FP32 Int8` Sample
```

Comment:
`Intel® AI Reference models for TensorFlow* Inference With FP32 Int8` Sample

Reply:
addressed it accordingly